src/pages/Login/index.tsx

```diff
@@ -156,16 +156,16 @@ const LoginPage: React.FC = () => {
             />
           </LeftContainer>
         ) : (
-          <div>
-            {renderForm()}
-            <BackgroundImage
-              src={Coders}
-              top={'60px'}
-              right={'0px'}
-              imgHeight={'100%'}
-            />
-          </div>
-        )
+            <>
+              {renderForm()}
+              <BackgroundImage
+                src={Coders}
+                top={'60px'}
+                right={'0px'}
+                imgHeight={'100%'}
+              />
```

Comment:
i managed to fix the scroll using a height of 'calc(100% - 60px)', also the image seems to be below the background

Reply:
Fix in #872